test/torchaudio_unittest/functional/autograd_cpu_test.py

```diff
@@ -0,0 +1,11 @@
+import torch
+from .autograd_impl import Autograd
+from torchaudio_unittest import common_utils
+import platform
+import unittest
+
+
+@unittest.skipIf(platform.system() == 'Windows', "not supported on windows")
```

Comment:
@yoyololicon We have landed https://github.com/pytorch/audio/pull/1345, and this should work on Windows too. Can you try removing this skip decorator?